server/src/test/java/org/spine3/server/entity/AbstractEntityRepositoryShould.java

```diff
@@ -105,6 +112,58 @@ public void handle_wrong_passed_ids() {
         }
     }
 
+
+    @SuppressWarnings({ "MethodWithMultipleLoops", "unchecked"})
+    @Test
+    public void retrieve_all_records_with_entity_filters_and_field_mask_applied() {
+        final EntityRepository<?, E, ?> repo = repository();
+
+        final int count = 10;
+        final List<E> entities = entities(count);
+
+        for (E entity : entities) {
+            repo.store(entity);
+        }
+
+        final List<EntityId> ids = new LinkedList<>();
+        for (int i = 0; i < count / 2; i++) {
```

Comment:
Please comment this piece of code. It's not absolutely clear, what you want to achieve.